### PR TITLE
Hearing Worksheet move autosave above canvas

### PR DESCRIPTION
**Pull request by nolawi**
#4230 
This PR doesn't change any functionality, Just moves the location of the Autosave component. 
Most of the changes are moving the redux stuff from `WorksheetHeader` to `HearingWorksheet`

screen shot 2018-01-04 at 4 37 53 pm
screen shot 2018-01-04 at 5 06 03 pm

### CODE REVIEW

File: app/assets/stylesheets/_hearings.scss [hunk 1]
```diff
@@ -447,6 +432,12 @@ $color-hearings-saving: #777;
 }
 
 @media print {
+  h1,
+  h2,
+  h3 {
+    font-family: sans-serif;
+  }
+
```
Comment (nolawi) [inline app/assets/stylesheets/_hearings.scss]: This will resolve font loading issues on PDF headers

Comment (sharonwarner) [reply app/assets/stylesheets/_hearings.scss]: 👍

File: client/app/hearings/HearingWorksheet.jsx [hunk 2]
```diff
@@ -116,6 +130,14 @@ export class HearingWorksheet extends React.PureComponent {
     });
 
     return <div>
+      {!this.props.print &&
```
Comment (sharonwarner) [inline client/app/hearings/HearingWorksheet.jsx]: I think we can get rid of the !this.props.print because we don't print outside the canvas anyway.

Comment (nolawi) [reply client/app/hearings/HearingWorksheet.jsx]: screen shot 2018-01-05 at 11 56 37 am
Got it, looks like we need it.

**Pull request merged**